### PR TITLE
Why u no merge??? 😡

### PR DESCRIPTION
**Pull request by CamSoper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # .NET 💜 GitHub Actions
 
+## Cam wuz here
+
```
Comment (IEvangelist) [inline README.md:4]:
```suggestion
```

No thanks Cam

File: Blazor.ClientApp/Blazor.ClientApp.csproj [hunk 2]
```diff
@@ -11,5 +11,5 @@
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="6.0.2" PrivateAssets="all" />
     <PackageReference Include="Blazor.LocalStorage.WebAssembly" Version="1.0.2" />
   </ItemGroup>
-
+<!-- Cam wuz also here -->
```
Comment (IEvangelist) [inline Blazor.ClientApp/Blazor.ClientApp.csproj:14]:
```suggestion
```

No thanks Cam

File: Blazor.ClientApp/Pages/Index.razor [hunk 3]
```diff
@@ -10,7 +10,7 @@
     <BotGallery ImageNames="_imageNames" />
     <br>
     <blockquote class="blockquote text-center pop-box shadow-lg large-text dotnet-purple">
-        <p>Get inspired by these .NET bot images!</p>
+        <p>Get pumped 💪🏻!!! .NET 💜 HYPE!!!11!!!</p>
```
Comment (IEvangelist) [inline Blazor.ClientApp/Pages/Index.razor:13]:
```suggestion
        <p>Get pumped .NET 💜 HYPE!!</p>
```

**Review by IEvangelist — DISMISSED** [review]
With a few nits. Otherwise let's :shipit:

File: README.md (hunk 1 above)
Comment (CamSoper) [reply README.md:4]: You're no fun 😝

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**